tidb-cloud/tidb-node-group-management.md

```diff
@@ -30,9 +30,13 @@ Currently, the TiDB Node Group feature is only available upon request. To reques
 
 ## Prerequisites
 
-- You have a [TiDB Cloud Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-cloud-dedicated) cluster deployed on AWS.
+- You have a [TiDB Cloud Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-cloud-dedicated) cluster deployed on AWS or Google Cloud.
 - You are in the **Organization Owner** or **Project Owner** role of your organization. For more information, see [User roles](/tidb-cloud/manage-user-access.md#user-roles).
 
+> **Note**:
+>
+> The TiDB node groups cannot be created during cluster creation. You need to add the groups after the cluster is created and in the **Available** state. 
+
```

Comment:
@qqqdan Should we include a link to the [Connect to a TiDB Cloud Dedicated Cluster via Google Cloud Private Service Connect](https://docs.pingcap.com/tidbcloud/set-up-private-endpoint-connections-on-google-cloud/) document in this step?
https://github.com/pingcap/docs/blob/7ac6502a7e7eb003451df5a06bf03fcc08963433/tidb-cloud/tidb-node-group-management.md?plain=1#L97